src/onepasswordconnectsdk/client.py

```diff
@@ -279,15 +258,15 @@ def update_item(self, item_uuid: str, vault_id: str, item: Item):
         item.id = item_uuid
         item.vault = ItemVault(id=vault_id)
 
-        response: requests.Response = self.build_request("PUT", url, item)
+        response = self.build_request("PUT", url, item)
```

Comment:
I wonder... Is there a reason for removing the expected type here?

Reply:
I see that `httpx` library supports both sync and async clients, which was the missing piece when looking at this code the first time. It makes sense to use the more suitable library in this case, which is `httpx` and drop the other one.